doc/write_your_own/components/Workflow1.ipynb

```diff
@@ -405,7 +408,20 @@
    }
   },
   "widgets": {
-   "state": {},
+   "state": {
+    "8074bd6a89a14444bf5f2966eebfee20": {
+     "views": []
+    },
+    "896b838d96d24f5fbf1fb99e2d9d5c0b": {
+     "views": []
+    },
+    "945eec722a2c47b183dde753cb49d909": {
+     "views": []
+    },
+    "dd3756b52eaa4a16bd8b19e748ca50aa": {
+     "views": []
+    }
+   },
```

Comment:
Done everything I can to remove it.  You want me to just delete it by hand? Seems risky...